src/plugins/embeddable/public/lib/embeddables/embeddable_factory.ts

```diff
@@ -81,7 +81,7 @@ export interface EmbeddableFactory<
    * Returns a display name for this type of embeddable. Used in "Create new... " options
    * in the add panel for containers.
    */
-  getDisplayName(): string;
+  getDisplayName(): JSX.Element | string;
```

Comment:
The `getDisplayName` changes are all related to allowing the names to also accept JSX, rather than just strings. In some situations we can now pass icons into the title, enhancing functionality.